[2025-08-24T04:25:24.187+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: poc_pipeline.process_upload manual__2025-08-24T04:25:20.694380+00:00 [queued]>
[2025-08-24T04:25:24.197+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: poc_pipeline.process_upload manual__2025-08-24T04:25:20.694380+00:00 [queued]>
[2025-08-24T04:25:24.197+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-08-24T04:25:24.210+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): process_upload> on 2025-08-24 04:25:20.694380+00:00
[2025-08-24T04:25:24.218+0000] {standard_task_runner.py:60} INFO - Started process 191 to run task
[2025-08-24T04:25:24.230+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'poc_pipeline', 'process_upload', 'manual__2025-08-24T04:25:20.694380+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/poc_pipeline.py', '--cfg-path', '/tmp/tmpzrps16jx']
[2025-08-24T04:25:24.233+0000] {standard_task_runner.py:88} INFO - Job 17: Subtask process_upload
[2025-08-24T04:25:24.272+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-08-24T04:25:24.355+0000] {task_command.py:423} INFO - Running <TaskInstance: poc_pipeline.process_upload manual__2025-08-24T04:25:20.694380+00:00 [running]> on host 046d27fa1875
[2025-08-24T04:25:24.534+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='poc_pipeline' AIRFLOW_CTX_TASK_ID='process_upload' AIRFLOW_CTX_EXECUTION_DATE='2025-08-24T04:25:20.694380+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-24T04:25:20.694380+00:00'
[2025-08-24T04:25:24.536+0000] {logging_mixin.py:188} INFO - [Dummy Mode] No real upload found. Using generated upload_id=dummy-577806af-cd9e-47ad-aea5-71bc309b9b34
[2025-08-24T04:25:24.537+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-08-24T04:25:24.563+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=poc_pipeline, task_id=process_upload, execution_date=20250824T042520, start_date=20250824T042524, end_date=20250824T042524
[2025-08-24T04:25:24.619+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-24T04:25:24.664+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
